js/src/ui/GasPriceEditor/store.js

```diff
@@ -104,6 +104,9 @@ export default class GasPriceEditor {
   }
 
   @action setConditionDateTime = (time) => {
+    time.setMilliseconds(0); // ignored by/not passed to Parity
```

Comment:
Potentially dangerous as it mutates `time` instead of cloning it using `new Date(time)`.